python/create_service_diagnostics_bundle.sh

```diff
@@ -72,12 +74,14 @@ function container_run () {
          -v "$(pwd)/${BUNDLES_DIRECTORY}:${CONTAINER_BUNDLES_DIRECTORY}" \
          -v "${HOST_DCOS_CLI_DIRECTORY}:${CONTAINER_DCOS_CLI_DIRECTORY_RO}":ro \
          ${CONTAINER_DCOS_SERVICE_DIAGNOSTIC_VOLUME_MOUNT} \
-         -e PYTHONPATH=${CONTAINER_DCOS_SERVICE_DIAGNOSTICS_DIRECTORY}/dcos-commons/testing:${CONTAINER_DCOS_SERVICE_DIAGNOSTICS_DIRECTORY} \
+         ${CONTAINER_DCOS_SERVICE_DIAGNOSTIC_DIR_ENV} \
+         -e DCOS_CLI_AUTO_INIT="${DCOS_CLI_AUTO_INIT}" \
          "${DOCKER_IMAGE}" \
          sh -l -c "${command}"
 }
 
 function usage () {
+  DCOS_CLI_AUTO_INIT=""
```

Comment:
Is this needed? `usage` will exit after the `container_run` call.

Also, please check my comment about this environment variable: https://github.com/mesosphere/dcos-sdk-service-diagnostics/pull/15/files#r327060595